_sass/components/_list.scss

```diff
@@ -9,24 +20,24 @@ ul a {
 
     &::before {
       color: $teal;
-      content: '•';
+      content: "•";
```

Comment:
You got me. Definitely a linter edit. I'm using **prettier** locally, which fortunately can be applied to many file types, including `scss` and `yaml`. I did notice we don't have any linting set up and have it in my list for separate efforts 😉 . I suggest implementing **prettier** as either a github action or a CircleCI step. Let me know your thoughts.

Reply:
We use Prettier for JavaScript in identity-idp, but I don't have a ton of experience with it for CSS. I know that, similar to what you've encountered, I had to go out of my way to configure my editor to _not_ apply it to all file types.

```
{
  "[javascript]": {
    "editor.formatOnSave": true
  },
  "[javascriptreact]": {
    "editor.formatOnSave": true
  },
  "[json]": {
    "editor.formatOnSave": true
  }
}
```

I like the idea of using Prettier, though ideally if it could be something we could somehow pair with automated linting to make sure the Prettier styling is applied consistently. In identity-idp, we use the ESLint Prettier plugin to error if a file isn't styled per Prettier expectations. Do you know if there's anything equivalent for (S)CSS?